components/pagination/stories/pagination.stories.js

```diff
@@ -1,9 +1,11 @@
 import { default as ActionButton } from "@spectrum-css/actionbutton/stories/actionbutton.stories";
 import { version } from "../package.json";
-import { PaginationGroup } from "./template";
+import { PaginationGroup, Template } from "./template";
 
 /**
  * The pagination component displays numbered buttons or an input field to allow for navigation.
+ * 
+ * The default listing/page variant uses buttons for each page number.  
```

Comment:
This is kind of a nitpick at this point and I don't have a strong opinion, but how would you feel about putting this as a comment on the Default story rather than the component overview?

Reply:
Thanks, I've moved that to the Default story.